root/static/scripts/edit/URLCleanup.js

```diff
@@ -232,6 +226,35 @@ const LINK_TYPES = {
   }
 };
 
+const RESTRICTED_LINK_TYPES = _.reduce([
```

Comment:
Name is fine, I think just linking to that page in a comment should be explanatory enough.